test/fixture/terraform_12/config.yaml

```diff
@@ -5,12 +5,10 @@ suites:
   - aws_tags_map
```

Comment:
remove some tests in here as well.
compatibility issue with existing providers. Not worth the effort to fix (0.12 is an old version).